tests/build_env_test.py

```diff
@@ -190,9 +192,9 @@ def test_env_validate(mocker, capsys):
         side_effect=buildTracker.validate_build_feedstock
     )
     env_file = os.path.join(test_dir, 'test-env-invalid1.yaml')
-    assert build_env.build_env([env_file]) == 1
-    captured = capsys.readouterr()
-    assert "Unexpected key chnnels was found in " in captured.err
+    with pytest.raises(OpenCEError) as exc:
+        build_env.build_env([env_file])
+    assert "Unexpected key chnnels was found in " in str(exc.value)
```

Comment:
typo `chnnels was`

Reply:
Please ignore the comment, as @npanpaliya pointed out that this is to test the incorrectness itself.